### DISCUSSION TITLE
Where is the documentation for @now/next, @now/static-build, @now/node, ...?

### DISCUSSION THREAD

**Discussion by piavgh** [vercel/vercel#3987] [General] [answered]
Hi everyone,

This might be a silly question but I can not find the documentation for these things anywhere, even after searching Google.

In next.js documentation or zeit.co docs, it only tells you to put code in`pages` folder or `pages/api` or `api` and then everything is configured **automatically**

I'm having a problem with these things (https://github.com/zeit/now/discussions/3985) so I want to understand it fully.

Thanks.

**Comment by paulogdm** [comment, marked as answer]
Hello,

We never recommending those packages directly. [Zero Config](https://zeit.co/blog/zero-config) is always the recommended approach.
Please check [this link](https://zeit.co/docs/runtimes#official-runtimes) for more information about runtimes.
